picard/formats/asf.py

```diff
@@ -138,6 +138,7 @@ class ASFFile(File):
         'djmixer': 'WM/DJMixer',
         'mixer': 'WM/Mixer',
         'artists': 'WM/ARTISTS',
```

Comment:
A little bit of internet research suggests that in WMP 11 and below they were case insensitive, but in WMP they are case sensitive. I suspect that Jaikoz created the tag first and we then adopted it with the case defined, but that may be wrong. Regardless, the case is now fixed by historical usage.